packages/react/src/components/DashboardEditor/DashboardEditorCardRenderer.test.jsx

```diff
@@ -211,7 +214,7 @@ describe('DashboardEditorCardRenderer', () => {
         isResizable
         key="listCard"
         title="ListCard render"
-        size={CARD_SIZES.SMALL}
+        size={CARD_SIZES.MEDIUM}
```

Comment:
SMALL is an unsupported size for ListCard. Instead of capturing the error (this happens in other tests) I just updated the size.